spec/emitter_spec.rb

```diff
@@ -729,9 +728,45 @@
         "thread=c at=start\n",
         "line=3\n",
         "thread=c at=finish elapsed=0.0000\n"
-      )
+      ) # the line "thread=e line=6" is not here b/c that was in a `#silence` block
 
       expect(other_lines).to contain_exactly("thread=d line=5\n")
     end
+
+    describe "#silence!" do
+      after do |example|
+        emitter.unsilence!
+      end
+
+      it "only affects the current thread" do
+        emitter.silence!
+
+        1_000.times do
+          emitter.log(thread: :main, line: 1)
```

Comment:
What do you think about naming this thread something like "silenced"? It can be helpful to better understand the intention of the block

Reply:
Yeah, I like that.